spec/factory_girl/registry_spec.rb

```diff
@@ -57,4 +57,19 @@
     subject.clear
     expect(subject.count).to be_zero
   end
+
+  it "removes a factory" do
+    subject.register(:object_name, registered_object)
+    expect do
+      expect(subject.remove(:object_name)).to eq(registered_object)
```

Comment:
is the significance of the return value that it found the factory? Are there cases where this value would be further used throughout the code?